src/components/bootcamp/search/email.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import * as yup from 'yup'
+import { FormikHelpers } from 'formik'
+import { Grid, Typography } from '@mui/material'
+import { SearchFormData } from 'gql/search'
+import GenericForm from 'components/common/form/GenericForm'
+import GenericGrid from '../utils/Grid'
+import SubmitButton from 'components/common/form/SubmitButton'
+import FormTextField from 'components/common/form/FormTextField'
+import BootcampersLayout from '../layout/Layout'
+import { axios } from 'common/api-client'
+import theme from '../layout/theme'
+
+const validationSchema: yup.SchemaOf<SearchFormData> = yup.object().defined().shape({
+  keyword: yup.string().required(),
+})
+
+const defaults: SearchFormData = {
+  keyword: '',
+}
+
+export type SearchFormProps = { initialValues?: SearchFormData }
+
+export default function SearchByPhone({ initialValues = defaults }: SearchFormProps) {
+  const [res, setRes] = React.useState([])
+
+  const onSubmit = async (values: SearchFormData, { resetForm }: FormikHelpers<SearchFormData>) => {
+    try {
+      setRes([])
+      const data = await axios.get(
+        'http://localhost:5010/api/bootcamp/search/email/' + values.keyword,
```

Comment:
Links like that are way better when we manage them in a single location like `routes.ts` and use them as functions.

This helps us refactor the URL more easily later on